### ISSUE TITLE
style_formats selector issue

### ISSUE THREAD

**Issue by manu37** [tinymce/tinymce#4797]
**Do you want to request a *feature* or report a *bug*?**
don't know
**What is the current behavior?**
In tinymce4 when a style_formats group is empty/not selected the group wasn't shown in styleselect, in tinymce5 an empty group is shown. bug or feature?
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
```
  style_formats:[
    {title: 'images', items: [
        {title: 'Styled image (left)',
            selector: 'img',
            classes: 'img-left'
        },
        {title: 'Styled image (right)',
            selector: 'img',
            classes: 'img-right'
        },
        {title: 'Styled image (center)',
            selector: 'img',
            classes: 'img-center'
        }
    ]}]
```
**What is the expected behavior?**
if there is no image selected it would be convenient to hide the whole group in styleselect as it does in tinymce4.

**Comment by lnewson** [comment]
I assume you mean when `style_formats_autohide` was enabled? as in v4 the menu item isn't removed/hidden by default unless that's enabled. If so, then here's a basic example to reproduce the issue: http://fiddle.tinymce.com/MBgaab

**Comment by wonderbugtiny** [comment]
This should now be fixed in 5.0.3. Please let us know if you find any further problems!
